go_covert_lib/CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing
+
+When making changes, please first either find the corresponding JIRA issue or create it if it does not exist.
+
+DO NOT COMMIT TO MASTER, please branch and PR any changes to master. This gives an opportunity for others to both code review your work and also see changes going into master, as opposed to new code simply appearing.
+
+## Branch Naming
+
+As you now have a JIRA issue tied to your changes, name your branch after your JIRA ticket number, and optionally a word or two after that describing the ticket.
+
```

Comment:
We should also make it so every commit message must have a JIRA ticket within the message. When we integrate JIRA with jithub it will make it easy as each commit will connect to a JIRA ticket

Reply:
New proposal: Anyone who says "Jithub" is forcefully evicted from the project

Reply:
But yeah, that's a good point